.goreleaser.yml

```diff
@@ -0,0 +1,61 @@
+before:
+  hooks:
+    - make release
+builds:
+  -
+    id: capp
+    env:
+      - CGO_ENABLED=0
+    binary: manager
+    goos:
+      - darwin
+      - linux
+    goarch:
+      - amd64
+      - arm64
```

Comment:
Will this try to build also for `darwin/arm64`? Will that even work?

Reply:
No, it is smart enough to build only what Go can build apparently 👍 in my test locally it built:

```
capp_darwin_amd64
capp_linux_amd64
capp_linux_arm64
```

Reply:
OK